engine/src/main/java/org/hibernate/validator/internal/xml/ClassLoadingHelper.java

```diff
@@ -49,11 +48,7 @@
 	private ClassLoadingHelper() {
```

Comment:
Yes, I guess so. I never was a fan of all these static methods in the XML package to begin with, but I didn't want to alter your design :) I can change it.